.gitattributes

```diff
@@ -1,5 +1,6 @@
 /.github/                       export-ignore
 /.phive/                        export-ignore
+/test/                          export-ignore
```

Comment:
When you work on a package and declare test code in separate directories with separate configuration files, adding exclusions for test code in `.gitattributes` is less work.

There is no need to add configuration files as they are in the excluded directory.